types/voilab-pdf-table/index.d.ts

```diff
@@ -0,0 +1,221 @@
+// Type definitions for voilab-pdf-table 0.5
+// Project: https://github.com/voilab/voilab-pdf-table
+// Definitions by: Daniel Sousa <https://github.com/danielb7390>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="pdfkit" />
+
+export = VoilabPdfTable;
+export as namespace VoilabPdfTable;
+
+declare class VoilabPdfTable<T = any> {
```

Comment:
Is the `any` the best choice here? It looks like non-base data type IMO

Reply:
Can you explain better?

Reply:
We would like to avoid people doing simple typo kind of thing with literals for e.g. number? this could be 'unknown` (as per handbook) or something that reassemble the type used (note: I've not used this native package, but I understand it accepts structure objects, so `Record<string, Type>` where type is allowed type). Instead of `Record` one could use `interface Type { [key: string]: SupportedType };`. 
What I meant: would be nice to guide what's allowed, but that's ok to use `unkown` (as per handbook)

Reply:
```suggestion
interface VoilabPdfTableDefaultType { [key: string]:  string | number};
declare class VoilabPdfTable<T = VoilabPdfTableDefaultType > {
```
If i understand you correctly, you are suggesting something like this? 
Should i also be checking that whatever type the user passes, the object values are always `string|number`?
I apologize if my questions are a bit weird, but i'm still learning TS

Reply:
Yes, if it make sense, I'll approve anyway. You can modify this further